stratum/glue/logging.cc

```diff
@@ -71,3 +71,15 @@ void InitStratumLogging() {
 }
 
 }  // namespace stratum
+
+// ostream overload for std::nulptr_t for C++11
+// see: https://stackoverflow.com/a/46256849
+#if __cplusplus == 201103L
+#include <cstddef>
+#include <iostream>
+namespace std {
+::std::ostream& operator<<(::std::ostream& s, ::std::nullptr_t) {
+    return s << static_cast<void *>(nullptr);
+}
+}
+#endif  // __cplusplus
```

Comment:
do we need this if we already defined in `logging.h` file?

Reply:
Look closely, logging.h contains the function declaration, in logging.cc is the function definition.
It has to be separated or else we would get ODR violations in every file that includes logging.h

Reply:
I see, thanks